contracts/auctions/FungibleAuction.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+// TODO: Use interface. The current version of @openZeppelin/contracts lib has none that fits.
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+// TODO: Switch to proper interface?
+interface IAuctionHouse {
```

Comment:
Personal preference, external interface file and events defined in the interface as they are part of the interface

Reply:
I like your preference. Will move some stuff around to align.